src/test/resources/application.yml

```diff
@@ -39,7 +39,8 @@ logging:
     org.springframework.web.reactive.function.client: info
     reactor.netty: info
     reactor: info
-    de.medizininformatikinitiative.torch: ${LOG_LEVEL:info}
```

Comment:
@Lucas0T  check if you can make the hapi client lib log level configurable and make it settable here